src/tests/stdlib/strpbrk.c

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file
+ * @brief Tests for strpbrk().
+ *
+ * @date Mar 30, 2020
+ * @author: Neeraj Adhikari
+ */
+
+#include <string.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("strpbrk test suite");
+
+TEST_CASE("strpbrk with empty strings") {
+	char nonempty[] = "test";
+	char empty[] = "";
+
+	test_assert_null(strpbrk(nonempty, empty));
+	test_assert_null(strpbrk(empty, nonempty));
+	test_assert_null(strpbrk(empty, empty));
+}
+
+TEST_CASE("strpbrk with printable strings") {
+	char embox[] = "EMBOXembox";
+	char vowels_lower[] = "aeiou";
+	char digits[] = "0123456789";
+	char symbols[] = "~!@#$%^&*()_+?><:}{\"'";
+	char nine[] = "9";
+	char hash[] = "#";
+
+	test_assert_equal(strpbrk(embox, embox), embox);
+	test_assert_equal(strpbrk(embox, vowels_lower), embox + 5);
+	test_assert_equal(strpbrk(vowels_lower, embox), vowels_lower + 1);
+	test_assert_equal(strpbrk(digits, nine), digits + 9);
+	test_assert_null(strpbrk(symbols, nine));
+	test_assert_equal(strpbrk(symbols, hash), symbols + 3);
+}
+
+TEST_CASE("strpbrk with non-printable data") {
+	char data[] = {0xCA, 0xFE, 0xBA, 0xBE, 0x00};
```

Comment:
maybe escape sequence is used here ([https://en.wikipedia.org/wiki/Escape_sequences_in_C](https://en.wikipedia.org/wiki/Escape_sequences_in_C))

Reply:
Used escape sequences inside quotation marks to encode non-printable strings.